Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -320,12 +320,22 @@ - (NSRange)selectedTextRange
 {
   return [super selectedRange];
 }
+
+- (NSArray *)readablePasteboardTypes
+{
+  NSArray *types = [super readablePasteboardTypes];
+  // TODO: Optionally support files/images with a prop
+  return [types arrayByAddingObjectsFromArray:@[NSFilenamesPboardType, NSPasteboardTypePNG, NSPasteboardTypeTIFF]];
```

Comment:
Dumb question, why would we want to support pasting an image into a Textinput? Why not just text?

Reply:
Ideally these becomes a configuration property, hence I raised this question in my summary

> Should we also add a property pasteTypes similar to drag&drag drag(ged)Types?

I have another pending change in which we further customize the supported paste types
https://github.com/christophpurrer/react-native-macos/commit/6c9e32d341c263276adf6eeb28cc6ced9158283f

I don't think we should or can hard code them in react-native-macOS.

-> Shall we add a property now?

Reply:
@Saadnajmi > If you agree with the overall approach, I am happy to add a configuration property for pasteTypes similar to drag&drag drag(ged)Types

Reply:
Sounds great.
I have another tiny change as well to follow up and then there is a bigger need for a property